src/host/_output.cpp

```diff
@@ -282,10 +287,32 @@ void WriteToScreen(SCREEN_INFORMATION& screenInfo, const Viewport& region)
         auto endingCoordinate = startingCoordinate;
         bufferSize.MoveInBounds(cellsModified, endingCoordinate);
         screenInfo.NotifyAccessibilityEventing(startingCoordinate.X, startingCoordinate.Y, endingCoordinate.X, endingCoordinate.Y);
+
+        // GH#3126 - This is a shim for powershell's `Clear-Host` function. In
+        // the legacy console, `Clear-Host` is supposed to clear the entire

```

Comment:
```suggestion
        // the traditional console, `Clear-Host` is supposed to clear the entire
```

NAK me if you want. I sorta feel like "legacy" is v1 and we don't have a name for v2 since Terminal came out so i've been calling it traditional

Reply:
"vintage" maybe? 😆